ha/util/conf_store.py

```diff
@@ -221,3 +221,20 @@ def get_disk_list_for_cvg(index, node_id, cvg_id):
         except Exception as e:
             Log.error(f"Unable to fetch Disk list from GConf. Error {e}")
             raise Exception(f"Unable to fetch Disk list. Error {e}")
+
+    @staticmethod
+    def get_control_pod(index):
+        """
+        Returns machine_id for control-node pod.
+        """
+        node_id = []
+        try:
+            node_key = Conf.search(index, GconfKeys.NODE_CONST.value,
+                                   GconfKeys.TYPE_CONST.value, GconfKeys.CONTROL_NODE.value)
```

Comment:
Are you using constant "Type" here?
Like, if type is "control_node"..!?

Reply:
> TYPE_CONST = "type"
CONTROL_NODE = "control_node"

These are the keys defined in const.py file.

Reply:
Added the example i/p & o/p in the func docstring for easy understanding.
Please check now.

Reply:
Thanks! As you have confirmed this from Archana. I am also fine with this change. Thanks!